src/game.cpp

```diff
@@ -5862,6 +5863,9 @@ void game::print_terrain_info( const tripoint &lp, const catacurses::window &w_l
     // if can't fit in one line.
     std::string tile = uppercase_first_letter( m.tername( lp ) );
     std::string area = uppercase_first_letter( area_name );
+    if( const timed_event *e = get_timed_events().get( timed_event_type::OVERRIDE_PLACE ) ) {
+        area = _( e->string_id );
```

Comment:
How can string_id be translated (that's what the `_()` does), it's an *identifier*, isn't it?

Reply:
Well yes and no, its probably a bad name because I added it to use as an id for mapgen and that kind of stuff.  Its just a string depending on the event type and in this case its the info to be displayed.